log-collector-script/EKS-ClusterCrawler.sh

```diff
@@ -0,0 +1,129 @@
+# #!/bin/bash
```

Comment:
I am not sure if this script should be reside inside an AMI?

Reply:
Agreed, we've looked at a similar script previously: https://github.com/awslabs/amazon-eks-ami/pull/886#issuecomment-1324119315

This type of resource should go in a dedicated troubleshooting repository, not the AMI repo.